extensions/cql/standard/clause_6_filter.adoc

```diff
@@ -7,29 +7,29 @@ include::requirements/requirements_class_filter.adoc[]
 
 <<OAFeat-1,OGC API - Features - Part 1: Core>> (and the draft http://docs.opengeospatial.org/DRAFTS/20-024.html[OGC API - Common - Part 2: Geospatial Data] standard) define two filtering parameters on the resource at path `/collections/{collectionId}/items`: `bbox` and `datetime`. <<OAFeat-1,OGC API - Features - Part 1: Core>> also adds support for simple equality predicates logically joined using the `AND` operator. These capabilities offer simple resource filtering for HTTP requests.
 
-This requirements class defines additional query parameters that allow more
+This Filter class defines additional query parameters that allow more
 complex filtering expressions to be specified when querying server resources.
 
 Specifically, this clause defines the parameters, `filter`, `filter-lang` and
 `filter-crs`.
 
-This requirements class also defines resources for discovering the list of
+The Filter requirements class also defines resources for discovering the list of
 resource properties (and their types) that may be used to construct filter
 expressions.
 
 [[filter-queryables]]
 === Queryables
 
-This extension does not assume that the content schema of a
-resource being queried is available for inspection and thus a means needs
-to exist to interrogate an endpoint to determine the names and types of the
+Part 3 does not assume that the content schema of a
```

Comment:
I would use "This document ..." or "This standard ...".

Reply:
Fixed.